src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedVideoConnection.java

```diff
@@ -76,31 +76,41 @@ public VersionedVideoConnection(VersionedLinkedInClient linkedinClient) {
     super(linkedinClient);
   }
   
-  public URN uploadVideoFromURL(InitializeUploadRequest initializeUploadRequest, String videoURL)
+  public URN uploadVideoFromURL(InitializeUploadRequest initializeUploadRequest, String videoURL,
+      boolean uploadThumbnail)
```

Comment:
Why do we add a separate parameter which we just set on the request further down? Can we not just rely on the user to set the field on the request directly?

Reply:
We can actually, didn't realise we can.